app/views/articles/new.html.erb

```diff
@@ -0,0 +1,13 @@
+<%= notice %>
```

Comment:
You are rendering the `notice` in application.html.haml and this will just duplicate the message

Reply:
done